docs-ref-conceptual/what-is-azure-cli.md

```diff
@@ -1,40 +1,60 @@
 ---
-title: What is Azure CLI?
-description: Overview of Azure CLI, a command-line interface tool designed to create and manage Azure resources now available in Windows, macOS, and Linux environments.
+title: What is the Azure CLI?
+description: Overview of the Azure CLI, a command-line tool designed to create and manage Azure resources available in Windows, macOS, Linux, and Docker containers.
 author: dbradish-microsoft
 ms.author: dbradish
 manager: barbkess
-ms.date: 01/30/2020
+ms.date: 06/04/2021
 ms.topic: overview
 ms.service: azure-cli
 ms.devlang: azurecli 
 ms.custom: devx-track-azurecli
 ---
-# What is Azure CLI
+# What is the Azure CLI
 
-The Azure command-line interface (Azure CLI) is a set of commands used to create and manage Azure resources.  The Azure CLI is available across Azure services and is designed to get you working quickly with Azure, with an emphasis on automation.
+The Azure CLI is a cross-platform command-line tool to connect to Azure and execute administrative commands on Azure resources. It allows the execution of commands through a terminal using interactive command-line prompts or a script.
 
-## Key Characteristics
+For interactive use, you first launch a shell such as cmd.exe on Windows, or Bash on Linux or macOS, and then issue a command at the shell prompt. To automate repetitive tasks, you assemble the CLI commands into a shell script using the script syntax of your chosen shell, and then you execute the script.
 
-Azure CLI capabilities make it easy to work with different programing languages and software environments.  For example, Azure CLI:
+You can install the Azure CLI locally on Linux, Mac, or Windows computers. It can also be used from a browser through the Azure Cloud Shell or run from inside a Docker container.
 
-- Is available to install in Windows, macOS, and Linux environments.
+## Syntax samples
```

Comment:
When you have a heading immediately following another heading, maybe one is unnecessary. 
You could make this info a bit more discoverable using two H2s and removing "Syntax samples":
## Subscription syntax example
## Role assignment syntax example

BTW: "Example" for code has much better search volume than Sample -- although I know we commonly use Sample at MSFT.